cppcache/src/TcrMessage.cpp

```diff
@@ -225,7 +544,7 @@ void TcrMessage::readCqsPart(DataInput& input) {
   // LOGDEBUG("mapsize = %d", m_cqs.size());
 }
 
-inline void TcrMessage::readCallbackObjectPart(DataInput& input,
```

Comment:
Why remove this inlining here?

Reply:
As per the spec:

"function specifiers, only allowed in function declarations"

This is not a function declaration.